requirements/development.txt

```diff
@@ -10,6 +10,14 @@ amqp==5.3.1
     # via
     #   -c requirements/base.txt
     #   kombu
+annotated-types==0.7.0
```

Comment:
i think "https://github.com/apache/superset/blob/master/scripts/uv-pip-compile.sh" had not been run for a while